packages/orn-locator/src/resolvers/books.ts

```diff
@@ -358,6 +382,28 @@ export const bookSearch = async(query: string, limit: number, filters: {[key: st
   return Promise.all(bookIds.map(book))
     .then(doSearch(query, limit));
 };
+const syncPageSearchData = (archiveBook: any) => (treeNode: any) => {
+  const bookId = archiveBook.id;
+  const pageId = treeNode.id.split('@')[0];
```

Comment:
I feel like those things should be in a shared content library. Maybe future work.

Reply:
Yea I'm not really happy with the code I wrote here, the orn-locator was supposed to only be recognizing the orn format and knowing where to look up the resource data. I ended up writing a bunch of code here because the content data is fragmented across osweb, archive and rex, so there isn't an easy place to get what I needed